scripts/setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+echo "Configuring System for Distributed Bitmap Engine"
+
+UPDATEFILE="/home/ubuntu/.dbe.checked-updates"
+DEPENDFILE="/home/ubuntu/.dbe.checked-dependencies"
+
+# Touch with `-a` ensures files exit without altering modification time.
+touch -a $UPDATEFILE
+touch -a $DEPENDFILE
+
+# We only run updates if it has been at least one day since we last checked.
+if test `find $UPDATEFILE -mmin +1440`
+then
+    echo "Ensuring System is Up to Date"
+    sudo apt-get --yes update
```

Comment:
if we're going to require the use of Ubuntu, shouldn't we check that if they aren't and print the issue, rather than just letting it fail

Reply:
Done.